x-pack/docs/en/rest-api/security/saml-logout-api.asciidoc

```diff
@@ -0,0 +1,77 @@
+[role="xpack"]
+[[security-api-saml-logout]]
+=== SAML logout API
+
+Submits a request to invalidate an access token and refresh token.
+
+[[security-api-saml-logout-request]]
+==== {api-request-title}
+
+`POST /_security/saml/logout`
+
+[[security-api-saml-logout-desc]]
+==== {api-description-title}
+
+This API invalidates the tokens that were generated for a user by the
+<<security-api-saml-authenticate,SAML authenticate API>>.
+
+If the SAML realm in {es} is configured accordingly and the SAML IdP supports
+this, the response to the request contains a URL to redirect the user to the IdP
+that contains a SAML logout request (starting an SP-initiated SAML Single Logout).
+
+{es} exposes all the necessary SAML related functionality via the SAML APIs.
+These APIs are used internally by {kib} in order to provide SAML based
+authentication, but can also be used by other custom web applications or other
+clients. See also <<security-api-saml-authenticate,SAML authenticate API>>,
+<<security-api-saml-prepare-authentication,SAML prepare authentication API>>,
+and <<security-api-saml-invalidate,SAML invalidate API>>.
+
+[[security-api-saml-logout-request-body]]
+==== {api-request-body-title}
+
+token`::
+(Required, string) The access token that was returned as a response to calling the
+  <<security-api-saml-authenticate,SAML authenticate API>>. Alternatively, the
+token that was received after refreshing the original one by using a
+  `refresh_token`.
```

Comment:
I think we should specify that it should be the _most recent_ token received for that session.